tests/Jellyfin.Model.Tests/Test Data/DeviceProfile-SafariNext.json

```diff
@@ -190,6 +190,28 @@
     ],
     "ContainerProfiles": [],
     "CodecProfiles": [
+        {
+            "Type": "Audio",
+            "Conditions": [
+                {
+                    "Condition": "LessThanEqual",
+                    "Property": "AudioChannels",
+                    "Value": "2",
+                    "IsRequired": false
+                }
+            ]
+        },
+        {
+            "Type": "VideoAudio",
+            "Conditions": [
+                {
+                    "Condition": "LessThanEqual",
+                    "Property": "AudioChannels",
+                    "Value": "2",
+                    "IsRequired": false
+                }
+            ]
+        },
```

Comment:
Is this how web going to supply global channel limit in the future or is it already the case?

Reply:
It is currently only set when `Settings -> Playback -> Maximum Audio Channels` is explicitly selected (not `Auto`).

Tbh, I think we should probably move this to `options.MaxAudioChannels` (since we don't have a special field in DeviceProfile) because it is not a codec limits.